charts/etcd-copy-backups/templates/etcd-copy-backups-job.yaml

```diff
@@ -232,6 +240,18 @@ spec:
           mountPath: "/root/.source-gcp/"
 {{- end }}
       volumes:
+{{- if eq .Values.targetStore.storageProvider "Local" }}
+      - name: target-host-storage
+        hostPath:
+          path: /etc/gardener/local-backupbuckets/{{ .Values.targetStore.storageContainer }}
```

Comment:
This path is hard-coded, so when it changes on g/g side it will break. Maybe it should be configurable?

Reply:
Any ideas how we can deliver this property from the `local-extension-provider` or the `gardnlet` to the druid in most elegant way without having to extend the `BackupBucket` or the `ETCD` objects just for the specific local stuff? Probably dumb idea, but we can add `bucketPath` to `etcd-backup` secret just for provider local. It would be something like:
```yaml
apiVersion: v1
data:
  bucketName: <bucket name>
  bucketPath: /etc/gardener/local or whatever the path would be
  kubeconfig: <kubeconfig>
  kind: Secret
....
```

Reply:
Yes, sounds like a good pragmatic idea!